packages/kotti-ui/source/kotti-comment/KtComment.vue

```diff
@@ -1,38 +1,38 @@
 <template>
-	<div class="comment">
+	<div class="kt-comment">
 		<KtAvatar size="sm" :src="userAvatar" />
-		<div class="comment__wrapper">
-			<div class="comment__info">
-				<div class="info__name" v-text="userName" />
-				<div class="info__time" v-text="createdTime" />
+		<div class="kt-comment__wrapper">
+			<div class="kt-comment__info">
+				<div class="kt-comment__info__name" v-text="userName" />
+				<div class="kt-comment__info__time" v-text="createdTime" />
 			</div>
 			<!-- eslint-disable vue/no-v-html -->
 			<div
 				v-if="!isInlineEdit"
-				class="comment__message"
+				class="kt-comment__message"
 				v-html="postEscapeParser(dangerouslyOverrideParser(inlineMessage))"
```

Comment:
Potentially out-of-scope, but this entire hacky feature can be replaced by using `white-space: pre-wrap`